quill-core/src/test/scala/io/getquill/ast/AstShowSpec.scala

```diff
@@ -505,4 +505,35 @@ class AstShowSpec extends Spec {
     (q.ast: Ast).show mustEqual
       """query[TestEntity].distinct"""
   }
+
+  "shows schema" - {
+    "table" - {
+      val q = quote {
+        query[TestEntity].schema(s => s.table("test"))
```

Comment:
I'd prefer to avoid a new explicit method, it could be confused with actual query composition. Would it be possible to implement something like this?

```scala
query[TestEntity](_.table("test"))
```

BTW, `table` isn't a good name as we could have sources that don't use tables, maybe just `name`?